app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb

```diff
@@ -0,0 +1,26 @@
+angular.module('formidable').controller('ModalChangeNameCtrl', function ($scope, $modalInstance, $http, $location, $routeParams, $rootScope) {
+
+  url = "<%= ENV['URL'] %>" || 'http://localhost:3000';
+  
+
+  $scope.changeFormName = function(newName) {
+    $scope.form = {
+      name: newName
+    };
+    $http.put(url + '/forms/' + $routeParams.id ,$scope.form).success(function(response) {
+       
```

Comment:
The behaviour we want is that when you change the name in the modal and click on 'change' the user can see the title of the form changed so there is no need for a confirmation pop up.